apps/financial-aid/web-veita/src/components/ModalTypes/AcceptModal.tsx

```diff
@@ -56,6 +56,10 @@ const AcceptModal = ({
   const router = useRouter()
   const maximumInputLength = 6
 
+  const showChildren =
+    applicationMunicipality.childrenAid === ChildrenAid.APPLICANT &&
+    hasApplicantChildren
```

Comment:
I would rename this variable since it is not only deciding logic about what is displayed.

I would probably go for something like hasChildrenAid